Os.py

```diff
@@ -0,0 +1,10 @@
+#python defined packages
+import sys
+import argparse
+import os
+import json
+
+#external packages
```

Comment:
Please make the document which shows how to install external packages

Reply:
Sure will create the manual